src/Icons/Bouncer.tsx

```diff
@@ -9,28 +9,12 @@ interface Props {
 
 const Bouncer: React.FunctionComponent<Props> = ({ width = '20', height = '24', fillColor = '#2D2926', ...rest }) => (
   <svg width={width} height={height} fill="none" viewBox="0 0 20 24" {...rest}>
-    <mask id="1yiv4evk9a" width="20" height="24" x="0" y="0" maskUnits="userSpaceOnUse">
-      <path
-        fill="#fff"
-        fillRule="evenodd"
-        d="M19 23h-1.83l-1.56-1.17c-1.208-.898-1.704-2.47-1.23-3.9.823-.425 1.579-.971 2.24-1.62 1.85-1.781 2.89-4.242 2.88-6.81 0-.191-.11-.365-.283-.448-.172-.083-.377-.06-.527.058-3.333 2.454-7.466 3.564-11.58 3.11-2.177-.19-4.192-1.227-5.61-2.89.086-4.438 3.569-8.065 8-8.33v2H7c-.276 0-.5.224-.5.5v2c0 .276.224.5.5.5s.5-.224.5-.5V4h5v1.5c0 .276.224.5.5.5s.5-.224.5-.5v-2c0-.276-.224-.5-.5-.5h-2.5V.5c0-.276-.224-.5-.5-.5-2.566.002-5.023 1.04-6.812 2.88C1.4 4.72.43 7.205.5 9.77c.137 3.418 2.08 6.506 5.1 8.11.481 1.43-.01 3.007-1.22 3.91L2.83 23H1c-.276 0-.5.224-.5.5s.224.5.5.5h18c.276 0 .5-.224.5-.5s-.224-.5-.5-.5zM1.62 10.91c1.518 1.296 3.395 2.098 5.38 2.3 4.018.44 8.066-.51 11.47-2.69-.53 4.371-4.309 7.616-8.71 7.48-4.039-.165-7.422-3.112-8.14-7.09zm5.13 7.49c.238 1.624-.434 3.248-1.75 4.23l-.5.37h10.99l-.49-.37c-1.305-.98-1.972-2.594-1.74-4.21-1.044.385-2.148.581-3.26.58h-.25c-1.026-.034-2.04-.237-3-.6z"
-        clipRule="evenodd"
-      />
-    </mask>
-    <g mask="url(#1yiv4evk9a)">
-      <path fill={fillColor} d="M-5 -3H25V27H-5z" />
-      <mask id="ipny5e689b" width="628" height="628" x="-615" y="-51" maskUnits="userSpaceOnUse">
-        <path fill="#fff" fillRule="evenodd" d="M-614.999-51H12.06v627.06H-615V-51z" clipRule="evenodd" />
-      </mask>
-      <g mask="url(#ipny5e689b)">
-        <path
```

Comment:
You may need to add this path (and perhaps the g) back in again. The first is what you changed it to, the second in the screenshot is the original
<img width="506" alt="Screen Shot 2020-10-05 at 2 36 21 PM" src="https://user-images.githubusercontent.com/17581359/95134731-3bded180-0718-11eb-93b2-99422ce10a82.png">

Reply:
what i see published and in figma is the first one with no fill for the base?

Reply:
the second one is the correct/original one

Reply:
then why isn't it like that on figma? https://www.figma.com/file/GAzzcQrMQnyE6NqQI8wX2P/DS---Upper-Funnel?node-id=0%3A1218